src/client/hooks/usePageChange.ts

```diff
@@ -60,9 +62,11 @@ export function usePageChange() {
                 strict: false,
               })
           )
-        ? 'Pagina niet gevonden'
+        ? NOT_FOUND_TITLE
         : PageTitleMain;
 
+    const pageFound = title !== NOT_FOUND_TITLE;
```

Comment:
```suggestion
    const isPageFound = title !== NOT_FOUND_TITLE;
```

Indien je een boolean variabele definieert, noem het dan: `isPageFound`, `hasError` etc.

Reply:
Var hernoemt